argh_derive/src/parse_attrs.rs

```diff
@@ -467,7 +467,7 @@ fn check_option_description(errors: &Errors, desc: &str, span: Span) {
         (Some(x), _) if x.is_lowercase() => {}
         // If both the first and second letter are not lowercase,
         // this is likely an initialism which should be allowed.
-        (Some(x), Some(y)) if !x.is_lowercase() && !y.is_lowercase() => {}
+        (Some(x), Some(y)) if !x.is_lowercase() && (y.is_alphanumeric() && !y.is_lowercase()) => {}
```

Comment:
`is_alphanumeric` and not `is_alphabetic` to support initialisms like I2C

Reply:
Seems to me that initialisms like I2C are already accepted:

```
!'2'.is_lowercase() = true
'2'.is_alphanumeric() && !'2'.is_lowercase() = true
```

Reply:
Yep, they are (even edited my message to say "still"). I'm just justifying my use of `is_alphanumeric` over `is_alphabetic`, which would prevent initialisms like I2C